src/nl/rubensten/texifyidea/settings/TexifyConfigurable.kt

```diff
@@ -18,6 +19,7 @@ class TexifyConfigurable(private val settings: TexifySettings) : SearchableConfi
     private lateinit var automaticUpDownBracket: JBCheckBox
     private lateinit var automaticItemInItemize: JBCheckBox
     private lateinit var automaticQuoteReplacement: ComboBox<String>
+    private lateinit var compilerCompatibility: ComboBox<String>
```

Comment:
Make the type `ComboBox<LatexCompiler>` to avoid checking the displayName (or does that break the serialization?)